### DISCUSSION TITLE
Species initialization by numpy array out of window.

### DISCUSSION THREAD

**Discussion by jiecaix** [SmileiPIC/Smilei#755] [Q&A] [answered]
I try to set position_initialization a numpy array which contains some particle positions out of window, and then open the moving window.

But i find that even the moving window had moved to the particle postion, it still no particle will be creation.

So i wonder if there any way to initialization particles out of windows (continues creation), either by "write a namelist" or "modification the source code".

**Comment by mccoys** [comment, marked as answer]
There is no way to do this currently.
